extensions/amp-auto-ads/0.1/amp-auto-ads.js

```diff
@@ -44,23 +50,28 @@ export class AmpAutoAds extends AMP.BaseElement {
       return;
     }
 
+    const extensionService = extensionsFor(this.win);
+    const ampAdPromise = extensionService.loadElementClass(AD_TAG);
+    const ampStickyAdPromise = extensionService.loadElementClass(STICKY_AD_TAG);
+
     const configPromise = this.getConfig_(adNetwork.getConfigUrl());
     const docPromise = this.getAmpDoc().whenReady();
-    Promise.all([configPromise, docPromise]).then(values => {
-      const configObj = values[0];
-      if (!configObj) {
-        return;
-      }
+    Promise.all([configPromise, docPromise, ampAdPromise, ampStickyAdPromise])
```

Comment:
I don't think processing of the config should be delayed on the ampAdPromise and ampStickyAdPromise being resolved. The amp auto ads stuff can still insert the ads and reserve the space before they load.